bigquery/README.md

```diff
@@ -24,6 +24,7 @@ if you haven't done so already.
     docker run -p 9200:9200 docker.elastic.co/elasticsearch/elasticsearch-oss:6.2.2
     ```
```

Comment:
Let's put deploy stuff in bigquery/deploy. This includes README.md and yaml.

Reply:
Done